row_027_bot.js

```
@@ -0,0 +1,362 @@
+            res.writeHead(200)
+            res.end("id needed xd")
+        }
+    } else {
+        res.sendFile('views/19.mp4', {
+            root: __dirname
+        })
+
+    }
+})
+app.get("/eval", function(req, res) {
+    if (!req.query.key) {
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)